tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -281,37 +287,43 @@ export const processDownload = async (
     parentFolder = folder !== 'undefined' ? folder : null
     downloads = await pageObject.download({
       folder: parentFolder,
-      names: files,
+      resources: files,
       via: actionType === 'batch action' ? 'BATCH_ACTION' : 'SIDEBAR_PANEL'
     })
+
+    downloads.forEach((download) => {
+      downloadedResources.push(download.suggestedFilename())
+    })
+
     if (actionType === 'sidebar panel') {
       expect(files.length).toBe(downloads.length)
-      downloads.forEach((download) => {
-        expect(files).toContain(download.suggestedFilename())
-      })
+      for (const resource of files) {
+        if (resource.type === 'file') {
+          expect(downloadedResources).toContain(resource.name)
+        } else {
+          // downloading folders in oc10 downloads with name of resource but in ocis it is downloaded as 'download.tar'
+          config.ocis
+            ? expect(downloadedResources).toContain('download.tar')
```

Comment:
previously we had this issue while checking with the extension https://github.com/owncloud/web/issues/7213 maybe we can drop it if it's not to important and only test the file name without the extension?

Reply:
Did not know about it thanks for the report. I will assert it without extension.